### DISCUSSION TITLE
How to enable brotli compression

### DISCUSSION THREAD

**Discussion by AnhTuan-AiT** [nginx-proxy/nginx-proxy#1778] [Q&A]
I wanted to change the default configuration file of nginx, so I created a Dockerfile using this image. Now, in this Dockerfile, how can I configure to enable **brotli compression**?

**Comment by junderw** [comment]
Modify the nginx.tmpl file. It's a template for generating nginx.conf files.

**Reply by AnhTuan-AiT** [reply]
@junderw Can you guide me with information about necessary configuration in that file?
